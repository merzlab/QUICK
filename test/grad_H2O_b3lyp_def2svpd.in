B3LYP  BASIS=def2-svpd cutoff=1.0e-9 denserms=1.0e-6  zmake GRADIENT DIPOLE CHARGE=0

 O     -0.33840     0.00380     0.23923
 H     -0.33510    -0.00190    -0.83277
 H      0.67350    -0.00190     0.59353


