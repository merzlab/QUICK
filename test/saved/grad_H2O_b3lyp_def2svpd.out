  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2021                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **                   This software provided pursuant to a license agreement containing                 **
  **                   restrictions on its disclosure, duplication, and use. This software               **
  **                   contains confidential and proprietary information, and may not be                 **
  **                   extracted or distributed, in whole or in part, for any purpose                    **
  **                   whatsoever, without the express written permission of the authors.                **
  **                   This notice, and the associated author list, must be attached to                  **
  **                   all copies, or extracts, of this software. Any additional                         **
  **                   restrictions set forth in the license agreement also apply to this                **
  **                   software.                                                                         **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Chi, J.; Cruzeiro, V.W.D.; Keipert, K.; Pekurovsky, D.; Mu, D.; Miao, Y.;He, X.; Ayers,K;
  Brothers, E.; GÃ¶tz, A.; Merz,K. M. QUICK-21.03 University of California San Diego, CA and Michigan State
  University, East Lansing, MI, 2021                                                                       
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Wed Mar 17 12:13:04 2021
| INPUT FILE :    grad_H2O_b3lyp_def2svpd.in
| OUTPUT FILE:    grad_H2O_b3lyp_def2svpd.out
| DATE FILE  :    grad_H2O_b3lyp_def2svpd.dat
| BASIS SET PATH: /scratch/cruzeiro/quick/basis/

 @ Read Job And Atom

  KEYWORD=B3LYP  BASIS=DEF2-SVPD CUTOFF=1.0E-9 DENSERMS=1.0E-6  ZMAKE GRADIENT DIPOLE CHARGE=0                                                                                                                    
 ============== JOB CARD =============
 METHOD = DENSITY FUNCTIONAL THEORY
 DENSITY FUNCTIONAL = B3LYP
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 READ SAD GUESS FROM FILE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 STANDARD GRID = SG1
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
      GRADIENT CUTOFF  =  0.100E-08
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = DEF2-SVPD,  TYPE = CARTESIAN
| BASIS FILE = /scratch/cruzeiro/quick/basis//DEF2-SVPD.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = O 
 BASIS FUNCTIONS =   25
 For Atom Kind =    2
 ELEMENT = H 
 BASIS FUNCTIONS =    8

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    3    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    2    NUMBER OF NON-HYDROGEN ATOM =    1
 NUMBER OF ELECTRONS     =   10

 -- INPUT GEOMETRY -- :
    O          -0.3384       0.0038       0.2392
    H          -0.3351      -0.0019      -0.8328
    H           0.6735      -0.0019       0.5935

 -- DISTANCE MATRIX -- :
              1           2           3
      1     0.00000
      2     2.02582     0.00000
      3     2.02607     3.30113     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   41
 NSHELL =   17 NPRIM  =   55
 JSHELL =   17 JBASIS =   27


 @ End Reading Molecular Information 


 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-09
   INITIAL GRID POINTS  =       11320
|   FINAL GRID POINTS    =        8283
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        7409
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =        8069

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1    -76.278396105      ------         0.235     1        0.01      0.22      0.01    0.1312E+01  0.3647E-01  0.3856E+00
|  2    -76.281882690  0.348659E-02       0.235     2        0.01      0.22      0.01    0.2692E+00  0.2597E-01  0.3209E+00
|  3    -76.151811161  -.130072E+00       0.234     3        0.01      0.22      0.01    0.4387E+00  0.1455E-01  0.2105E+00
|  4    -76.363593449  0.211782E+00       0.192     4        0.01      0.18      0.00    0.8148E-02  0.6125E-03  0.5058E-02
|  5    -76.363714254  0.120806E-03       0.184     5        0.01      0.17      0.00    0.2483E-02  0.1367E-03  0.1191E-02
|  6    -76.363723013  0.875881E-05       0.184     6        0.01      0.17      0.00    0.5135E-03  0.2259E-04  0.2981E-03
|  7    -76.363723297  0.284491E-06       0.185     7        0.01      0.17      0.00    0.4838E-04  0.2968E-05  0.3135E-04
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-08 ------------
|  8    -76.363723294  -.297355E-08       0.203     8        0.03      0.18      0.01    0.1582E-04  0.6605E-06  0.8392E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   8 CYLCES
| MAX ERROR = 0.158235E-04   RMS CHANGE = 0.660473E-06   MAX CHANGE = 0.839242E-05
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =    4.9999834707
 BETA ELECTRON DENSITY     =    4.9999834707
 ELECTRONIC ENERGY    =   -84.564195024
 CORE_CORE REPULSION  =     8.200471729
 TOTAL ENERGY         =   -76.363723294

 @ End Energy calculation


 @ Begin Gradient Calculation

 ANALYTICAL GRADIENT: 
----------------------------------------
 COORDINATE    XYZ            GRADIENT
----------------------------------------
    1X    -0.3384000258    -0.0603421379
    1Y     0.0038000003     0.0006663008
    1Z     0.2392300183     0.0422076431
    2X    -0.3351000256    -0.0088794152
    2Y    -0.0019000001    -0.0003344820
    2Z    -0.8327700636    -0.0761179441
    3X     0.6735000514     0.0692019374
    3Y    -0.0019000001    -0.0003360586
    3Z     0.5935300453     0.0338997665
----------------------------------------

 @ End Gradient Calculation


 @ Begin Zmake conversion

  
 Z-MATRIX:

   ATOM  ELEMENTAL  BOND LENGTH    BOND ANGLE    DIHEDRAL ANGLE
  NUMBER  SYMBOL    (ANGSTROMS)    (DEGREES)       (DEGREES)
     I                 I-NA         I-NA-NB        I-NA-NB-NC      NA   NB   NC

     1      O        0.00000  0     0.00000  0      0.00000  0      0    0    0
     2      H        1.07202  1     0.00000  0      0.00000  0      1    0    0
     3      H        1.07215  1   109.11820  1      0.00000  0      1    2    0

 @ End Zmake conversion


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   O             -0.4067            -0.1418
   H              0.2033             0.0691
   H              0.2034             0.0727
   TOTAL         0.0000             0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
    1.5311   -0.0172   -1.0825    1.8752

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     1.047265000( 31.38%)
| DFT GRID OPERATIONS =     0.060523000(  1.81%)
| TOTAL SCF TIME      =     1.706490000( 51.14%)
|       TOTAL OP TIME      =     1.607945000( 48.18%)
|             TOTAL 1e TIME      =     0.054176000(  1.62%)
|             TOTAL 2e TIME      =     0.808199000( 24.22%)
|             TOTAL EXC TIME     =     0.687723000( 20.61%)
|       TOTAL DII TIME      =     0.096113000(  2.88%)
|             TOTAL DIAG TIME    =     0.047022000(  1.41%)
| TOTAL GRADIENT TIME      =     0.528339000( 15.83%)
|       TOTAL 1e GRADIENT TIME      =     0.058310000( 1.80%)
|       TOTAL 2e GRADIENT TIME      =     0.270152000( 8.10%)
|       TOTAL EXC GRADIENT TIME     =     0.197892000(  5.93%)
| TOTAL TIME          =     3.337217000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  3.3 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Wed Mar 17 12:13:07 2021
